docs/integrations.md

```diff
@@ -1 +1,32 @@
 # Integrations
+
+## Apache Atlas API
+
+Apache Atlas offers a versatile REST API designed for managing metadata entities,
+types, lineage, and data discovery. This API provides endpoints for various operations
+including entity CRUD, search functionalities, glossary management, lineage information
+retrieval, and relationship management.
+
+For detailed documentation and examples, refer to the [Apache Atlas API Documentation](https://atlas.apache.org/api/v2/index.html).
+
+## Data Governance API
```

Comment:
By data governance API I meant the lineage rest API that gaby made. See link below. 

https://github.com/aureliusenterprise/aurelius/tree/main/backend/m4i-lineage-rest-api

What you have here are back end services that are not really meant for users to access directly. I will maybe mention them as part of the architecture but they do not need to be listed here.